src/.vuepress/components/ExtensionsList.vue

```diff
@@ -1,25 +1,93 @@
 <!--
-Create a bulleted list of sources from the external repository url.
-The component get the sources from the `versioning.json` file of the repo.
-The url argument must be of the form "https://paperback-ios.github.io/extensions" without a trailing slash.
+Create a table representing an external repository.
+The component gets the sources from the `versioning.json` file of the repo.
+The url argument must point to the webpage hosting the `versioning.json` file and must not have a trailing slash EX:https://paperback-ios.github.io/extensions.
 -->
 
 <template>
-	<div>
-		<ul id="v-for-object">
-			<li v-for="extension in extensions"
-			:id="extension.name"
-			:key="extension.id"
-			>
-				{{ getName(extension.name) }}
-				<span v-for="tag in extension.tags" :key="tag.text"> <el-tag :type="tag.type" size="mini" effect="dark">{{ tag.text }}</el-tag> </span>
-			</li>
-		</ul>		
-	</div>
+	<table>
+		<thead>
+			<tr>
+				<th><b>{{ name }}</b></th>
+				<th class="hidden-md-and-down">{{ url }}</th>
+				<th><a :href="encodedURL()">Add to Paperback</a></th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td colspan="3" class="hidden-lg-and-up"><b>{{ url }}</b></td>
+			</tr>
+			<tr>
+				<td colspan="3">{{ description }}</td>
+			</tr>
+			<tr>
+				<td colspan="3">
+					<ul id="v-for-object" class="extensionList">
+						<li
+							v-for="extension in extensions"
+							:id="extension.name"
+							:key="extension.id"
+						>
+						
+							<ElTooltip placement="bottom" v-if="extension.tags.length > 0">
```

Comment:
Currently the component breaks when the tag list is undefined.
`TypeError: can't access property "length", e.tags is undefined`

```suggestion
							<ElTooltip placement="bottom" v-if="extension.tags != undefined && extension.tags.length > 0">
```